tests/integration/Owncloud/OcisPhpSdk/ShareTestGetSharedWithMeTest.php

```diff
@@ -53,18 +56,28 @@ public function setUp(): void
 
     }
 
-    public function testGetShareWithMe(): void
+    public function testGetAttributesOfReceivedShare(): void
     {
-        $this->folderToShare->invite($this->einstein, $this->editorRole);
-        $receivedShare = $this->einsteinOcis->getSharedWithMe();
-        $this->assertInstanceOf(ShareReceived::class, $receivedShare[0]);
-        $this->assertEquals($this->folderToShare->getName(), $receivedShare[0]->getName());
-        $this->assertSame($this->folderToShare->getId(), $receivedShare[0]->getRemoteItemId());
-        //The step will only work after this bug is solved https://github.com/owncloud/ocis/issues/8000
-        //$this->assertSame($this->personalDrive->getId(), $receivedShare[0]->getDriveId());
+        $this->fileToShare->invite($this->einstein, $this->editorRole);
+        /**
+         * @var ShareReceived $receivedShare
+         */
+        $receivedShare = $this->getSharedWithMeWaitTillShareIsAccepted($this->einsteinOcis)[0];
+        $this->assertInstanceOf(ShareReceived::class, $receivedShare);
+        $this->assertMatchesRegularExpression('/' . $this->getUUIDv4Regex() . '/', $receivedShare->getRemoteItemId());
+        $this->assertMatchesRegularExpression('/' . $this->getFileIdRegex() . '/', $receivedShare->getId());
```

Comment:
Hm, all those IDs are supposed to be opaque. The fact that they are composed of UUIDs with some form of separators is an implementation detail. I am not exactly sure why a client implementation should test of that.

I haven't looked at the implementation of `getUUIDv4Regex()`, but there is something fishy about it. The `id` of the `remoteItem` is not a simple UUID. It's basically the same format as the driveItemId can be.

Reply:
Thanks, I've changed the assertions to only check that the ID is not an empty string